src/key-handler/index.tsx

```diff
@@ -61,11 +67,11 @@ export function KeyHandler(props: KeyHandlerProps) {
         );
       };
     },
-    [focusedStack, focusGroupId, handler, triggers]
+    [focusedStack, focusGroupId, rest, triggers]
   );
 
-  if (shouldRenderChildren) {
-    return <FocusGroup>{props.children}</FocusGroup>;
+  if (shouldRenderChildren && "children" in rest) {
+    return <FocusGroup>{rest.children}</FocusGroup>;
```

Comment:
This is fine, but we can also do a similar trick to make this a little less awkward:

```ts
const children = "children" in rest ? rest.children : null;
```

```suggestion
  if (shouldRenderChildren && children) {
    return <FocusGroup>{rest.children}</FocusGroup>;
```